pyproject.toml

```
@@ -23,3 +34,20 @@ before-all = [
 ]
 # Skip musllinux builds for the moment
 skip = "*-musllinux_*"
+
+[tool.scikit-build]
+metadata.version.provider = "scikit_build_core.metadata.setuptools_scm"
+cmake.version = ">=3.11.0"
```

3.15 is the first version scikit-build-core supports.

---

```suggestion
cmake.version = "CMakeLists.txt"
```

If you set the min correctly in CMakeLists.txt.